ttnn/cpp/ttnn/operations/experimental/paged_cache/paged_cache.hpp

```diff
@@ -0,0 +1,40 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "device/paged_cache_operation.hpp" // TODO: not right!
```

Comment:
You should avoid device operation include in the operation header. Think of it as a detail of implementation. 
if something from it is needed here it is best to have that extracted into <op>_types.hpp or <op>_common.hpp